### PR TITLE
build(deps-dev): bump rollup from 2.51.2 to 4.12.0

### PR DESCRIPTION
**Pull request by Fdawgs**

### CODE REVIEW

File: rollup.config.mjs [hunk 1]
```diff
@@ -45,11 +49,8 @@ export default [
       resolve(),
       json(),
       commonjs(),
-      babel({
-        babelrc: false,
-        presets: ['@babel/preset-env'],
-      }),
-      sizeCheck(),
+      babel(babelConfig),
+      // sizeCheck(),
```
Comment (Fdawgs) [inline rollup.config.mjs:53]: `rollup-plugin-filesize-check` throws `[!] (plugin filesize) TypeError: The "string" argument must be of type string or an instance of Buffer or ArrayBuffer. Received undefined` when handling sourcemaps, so have commented out for now.

Comment (spencermountain) [reply rollup.config.mjs:53]: cool, feel-free to remove filesize-check if it's causing problems. rollup is full of these sorta gotchas.

**Comment by Fdawgs** [comment]
Seems to be unhappy with node 20, grumble grumble 😖

**Comment by Fdawgs** [comment]
Test failure unrelated. Only occurs for Windows so I'm assuming some issue with LF vs CRLF?

**Comment by spencermountain** [comment]
Ya- your guess is as good as mine. Looks good to me- do you have a npm account? I can add you as a publisher.
Cheers

**Comment by Fdawgs** [comment]
Cheers @spencermountain, I do have an NPM account: https://www.npmjs.com/~fdawgs

**Comment by Fdawgs** [comment]
@spencermountain Would I also be able to be added as a maintainer for this repo, please? Means I can merge, cut releases, etc.

**Comment by spencermountain** [comment]
looks good to me! I've added you as a maintainer.
cheers

**Pull request merged**